packages/common/src/models/USDCTransactions.ts

```diff
@@ -0,0 +1,79 @@
+import { Nullable } from '../utils/typeUtils'
+
+import { StringUSDC } from './Wallet'
+
+export enum USDCTransactionType {
+  PURCHASE = 'PURCHASE',
```

Comment:
nitty nit: maybe `PURCHASE_USDC`, so `PURCHASE_CONTENT` doesn't seem like a subset of this?

and then, could `USDCTransactionMetadataType` go away? I know we're modeling this off the old `Audio` tx history stuff

Reply:
Man I really overloaded "Buy" "Purchase" etc in the buy audio flow.. I could go either way - either keeping it "PURCHASE" to match the Audio one, or what Piazza is suggesting to make an improvement despite losing consistency

Reply:
I'm all for PURCHASE_USDC. I think having the separate types was mostly just convention/semantics. So I'll ditch the separate metadata type enum and just use the transaction type as the `discriminator`